www/en/coding-standard.texy

```diff
@@ -206,7 +206,7 @@ Class members implementation should follow this order as it is practical:
   b) protected
   c) private
 
-The ultimate ancestor of all instantiable classes should be `Nette\Object`.
+All instantiable classes should use `Nette\SmartObject` or `Nette\StaticClass` traits.
 
```

Comment:
All instantiable classes should use `Nette\SmartObject` trait and non-instantiable `Nette\StaticClass` trait.